### ISSUE TITLE
[Core Mobile] Push notification for disability claim decision letter availability

### ISSUE THREAD

**Issue by rachelhanster** [department-of-veterans-affairs/va-mobile-app#8034]
# Epic Status Overview
### Problem Statement:
_High level description of the epic and the problem we are trying to solve_
As a Veteran using the mobile app with push notification setting on, I want to be informed that a disability claim decision letter is available. Upon clicking a push notification, I am routed directly to the letter (deep linking).

### Why is this Important?:

Reduce time and work for Veterans to get disability claim decision via mobile phone and mobile app. Inform Veterans even earlier that a disability claim decision letter is now available for view and download.

### Definition of Done:
What set of criteria must be met for this epic to be considered complete?

- User is able to enable a push notification to be sent to them when their disability claim decision letter is available to view. 
- User is able to click on the push notification and be routed to view their decision letter in the app. 
- User is able to dismiss notification without clicking or viewing.
- User is able to disable push notification is they do not want to receive the push notification.
- Currently only sending emails when a decision associated to one of the following end products (EPs) is made: 010, 110, or 020. By September, we plan to expand the scope to decisions for additional types of claims (e.g., pension, supplemental claims, higher level reviews). Mobile should follow what is being shared through email notifications.

### Success Criteria:
What measurable outcomes will be used to indicate that this epic has successfully delivered value (i.e. - what is the impact)?

- Veterans are more quickly made aware when their disability claim decision letter is ready for them to view. 
- Veterans view their decision letter through clicking on the push notification

### Product Brief:
Mobile app [push notification product brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/features/Push%20Notifications/product/Push%20Notification%20Product%20Brief.md)
Mobile app [claims reimagined product brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/va-mobile-app-sitemap/products/va-mobile-app/features/claims-discovery/Claims%20Reimagined%20Product%20Brief.md)

**Related documentation:**
Epic for the email notification: https://github.com/department-of-veterans-affairs/digital-experience-products/issues/1055
Event Bus for notification: https://github.com/department-of-veterans-affairs/va.gov-team/issues/77622
[Claim letters product guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/claim-appeal-status/product-guides/Claim%20Notification%20Letters%20Product%20Guide%20v1.2%20SRT%2006102025.docx)

### Monthly Update for Stakeholders

 
|Date|  Progress & Key Accomplishments| Issues/Risks/Blockers|
|-------| --------|---------|
|August 2024 | Green | Continuing to work with the BMT team and Event Bus
|08/25/2025| Submitted intake ticket with VA Notify. Content and event bus trigger for push are ready. 
|10/28/2025| Benefits team continuing work in this epic: https://github.com/department-of-veterans-affairs/va.gov-team/issues/121421. Closing #8034 and ongoing work will be initiated and monitored by Benefits team.
|MM/DD/YY|

### Current Dependencies / Blockers
<details>
<summary>Blockers</summary>

|Blocker | Team / Owner | Est to Resolution |
|-------|---------|-------| 
| Internal dependency ? | none  | none
| External dependency ?  | Event Bus and Benefits Integration Platform (BIP)  | blocked until April 2024. Event Bus team and BIP need to create an endpoint, a status letting the Veteran know they have a decision letter available. This will be in production for mobile app consumption ~April 2024. Alan Werner in Slack #ves-event-bus

### VA Stakeholders

- 
 
### Reference Links
List links related to epics, pitch deck, one pager, product folder, etc.

- VeText PO: Shane Elliott
- VA Notify PO: Dave Conlon
- Benefits PO: Amy Lai

**Comment by jstrothman** [comment]
Noting that unfortunately, the doc type 184 (Decision Notices) includes some other types of letters, e.g. Intent to File. It'd be more accurate to refer to these as, e.g. `Claim decision (or other notification, like Intent to File)`

**Comment by BeccaTupaj** [comment]
@rtwell Is this the right epic ticket to be looking at for the Push Notification for the change in claim status? 
cc: @timwright12

**Comment by rtwell** [comment]
No, I do not think so (or it is a sub-issue meaning this is one of the many changes that can happen to your claim).

**Comment by BeccaTupaj** [comment]
@rtwell Oh you are right! I think #6656 is meant to be the main epic ticket

**Comment by amylai-va** [comment]
@BeccaTupaj @rtwell Hello! The VA.gov and Event Bus teams are working toward a pilot launch for decision letter email notifications at the end of this month (June), and we'd like to discuss adding mobile app push notifications as a possible goal for the next quarter. Next week, I'll add more information to this ticket and would be happy to answer any questions after that. Thanks! cc/ @jstrothman @CorySohrakoffUSDS

**Comment by amylai-va** [comment]
@BeccaTupaj @rtwell Hello! This week, we began our release of email notifications to Veterans when a decision is made on their claim 🥳 (we had a successful canary release and are going up to 25% tomorrow)

Scope-wise, we are currently only sending emails when a decision associated to one of the following end products (EPs) is made: 010, 110, or 020. By September, we plan to expand the scope to decisions for additional types of claims (e.g., pension, supplemental claims, higher level reviews). We'd love for mobile app notifications to complement these emails and wondering if now would be a good time to loop you all in?

Here's some relevant documentation:
- [super epic - electronic notification of benefits decisions in support of PACT Section 807 and AR77](https://github.com/department-of-veterans-affairs/digital-experience-products/issues/1055)
- [epic - creating notification for when decision letter is available](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77622)
- [decision letter availability (DLA) Event Bus events](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Use%20Cases/Decision%20Letter%20Availability/Decision%20Letter%20Availability%20Use%20Case.md#triggering-the-decision-letter-availability-event)
- [Claim letters product guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/claim-appeal-status/product-guides/Claim%20Notification%20Letters%20Product%20Guide%20v1.2%20SRT%2006102025.docx)

**Comment by BeccaTupaj** [comment]
Thank you for sharing this @amylai-va ! I will review the relevant documentation, and sync up with @rtwell  to determine when our team has ability to prioritize this work.
